gen/emit/EmitJulia.cpp

```diff
@@ -366,8 +367,7 @@ struct EmitJulia : public Emitter
     {
         auto* sn = ls.structname.nameUnderscoreCStr();
 
-        emit(0, "import ZCM.encode");
-        emit(0, "function encode(msg::%s)", sn);
+        emit(0, "function ZCM.encode(msg::%s)", sn);
```

Comment:
Why not `import ZCM.encode` and then call this function `encode`?

Reply:
Doing that causes warnings about "ignoring duplicate import..." when including more than one zcmtype generated file.

Reply:
Ahh that makes sense